build.sh

```diff
@@ -16,12 +37,30 @@ echo ""
 echo "Which signing identity do you want to use?"
 echo "1. SURFnet B.V. (ZYJ4TZX4UU)"
 echo "2. Egeniq (E85CT7ZDJC)"
-echo "3. Other"
-read -p "0-9?" choice
+echo "3. Enter own Team(This isn't just a random name, but it must be exact as in their signing identity): "
```

Comment:
`3. Enter own Team(This isn't just a random name, but it must be exact as in their signing identity): "` --> `3. Other "`

Reply:
Hi Johan, So you want to change the text here to other?